decidim-consultations/app/controllers/decidim/consultations/admin/application_controller.rb

```diff
@@ -8,6 +8,10 @@ class ApplicationController < Decidim::Admin::ApplicationController
         layout "decidim/admin/consultations"
 
         helper Decidim::SanitizeHelper
+
+        def current_participatory_space_manifest_name
+          :consultations
```

Comment:
I know this was due to an unexpected issue, but I don't quite like it. If we can't do it this way, I'd find an alternative approach. This looks redundant and will harm the developer experience sooner or later.